resources/quality/mixware_vulcan/mixware_vulcan_draft.inst.cfg

```diff
@@ -0,0 +1,17 @@
+[general]
+version = 4
+name = Draft
+definition = mixware_vulcan
+
+[metadata]
+type = quality
+quality_type = draft
+global_quality = True
+setting_version = 20
+
+[values]
+layer_height = 0.15
+layer_height_0 = 0.18
+infill_before_walls = False
+support_infill_rate = 20
+acceleration_travel = 500
```

Comment:
These three settings are the same for all the profiles that you provided. So they aren't profile settings, they are machine defaults right? As a rule of thumb, profiles should only have the settings that they change as this makes it much easier to maintain in the future.